### DISCUSSION TITLE
Can't find my scenes or characters

### DISCUSSION THREAD

**Discussion by HYPE-STUDIOS-PHILIP** [mozilla/hubs#3031] [General Questions & Discussion] [answered]
Hi!

I made a custom scene and a custom character for me and others to use, but when i search for either the scene or avatar I can't find them. Is there something I'm doing wrong or is it not possible for others to find my scene for themselves?

Pushed it for Mozilla to promote

![Raadsalen](https://user-images.githubusercontent.com/34505958/93322288-6f15fb00-f813-11ea-8a6d-9ff232e32998.JPG)
![Raadsalen_avatar](https://user-images.githubusercontent.com/34505958/93322291-6fae9180-f813-11ea-8ed4-c95306a861ee.JPG)

**Comment by HYPE-STUDIOS-PHILIP** [comment]
Is it also possible to disable drawing, adding elements and teleport?

**Reply by emclaren** [reply]
Will answer this here https://github.com/mozilla/hubs/discussions/3037

**Comment by emclaren** [comment, marked as answer]
Hi there - for the moment, only you will have access to your avatar ( it won't show up in search.)

If you marked it as "promotable" we try to do content review about once a week (we test to make sure that the avatars aren't prohibitively large or broken somehow, and that they don't include anything violent/racist/offensive/identifying). That being said, this process can be a bit slower. I will make sure your avatar gets added to the search the next time I am in the Hubs content back-end.

I will create a new discussion topic for your second question because I think other folks will probably be looking for the answer as well!

**Reply by emclaren** [reply]
are there scenes you want to make sure get added as well?

**Reply by HYPE-STUDIOS-PHILIP** [reply]
Yes, If you could add "Raadsalen" that would be great! 

Thank you for answering so fast! :)

**Reply by HYPE-STUDIOS-PHILIP** [reply]
How long do you think it will take? We are going to use it for teaching and require it pretty soon :)

**Reply by rawnsley** [reply]
@HYPE-STUDIOS-PHILIP You can place avatar selectors into your scene without having to have them approved: https://github.com/mozilla/hubs/discussions/3038

**Reply by emclaren** [reply]
Hi there - just confirming that I've approved the avatar named  "Munk" at the start of the week. It should be discoverable in the "newest avatar" tab. I didn't see anything named "Raadsalen" in the list of avatars for approval, so please let me know if you think this may have been a mistake (feel free to email hubs@mozilla.com about content curation questions)
